bin/bump_version.py

```diff
@@ -142,7 +142,8 @@ def bump_version() -> None:
             "pre-commit",
             "run",
             "--files=docs/changelog.md",
-        ]
+        ],
+        check=True,
```

Comment:
```suggestion
        check=False,
```

This one is deliberately not asserted, as it typically has a non-zero exit code on the first run, when one of the pre-commit hooks makes a change to update the changelog in the README.

The second run afterwards is check'd, and that one we do care about.

Reply:
That's why I asked. Having it explicit helps make it clearer this is what is happening. :)

Reply:
agreed